google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/internal/TopicStatsClient.java

```diff
@@ -43,4 +44,17 @@ static TopicStatsClient create(TopicStatsClientSettings settings) throws ApiExce
    */
   ApiFuture<ComputeMessageStatsResponse> computeMessageStats(
       TopicPath path, Partition partition, Offset start, Offset end);
+
+  /**
+   * Compute the head cursor for the partition. The head cursor's offset is guaranteed to be before
+   * or equal to all messages which have not yet been acknowledged to be published, and greater than
+   * the offset of any message whose publish has already been acknowledged. It is 0 if there have
+   * never been messages on the partition.
+   *
+   * @param path The topic to compute head cursor on
+   * @param partition The partition to compute head cursor for
+   * @return A future that will have either an error {@link ApiException} or the
+   *     ComputeHeadCursorResponse on success.
+   */
+  ApiFuture<ComputeHeadCursorResponse> computeHeadCursor(TopicPath path, Partition partition);
```

Comment:
This response is a bit silly, and basically exists for aip reasons. Can you just return the Cursor instead?